frontend/pages/review.tsx

```diff
@@ -35,29 +37,88 @@ function ErrorWrapper({ message }: { message: string | undefined }) {
 }
 
 function ApprovalList({ category, filter }: { category: ApprovalCategory; filter: ApprovalFilterType }) {
```

Comment:
As we now have upload category and approval category, we should update this prop to be clearer. Change `category` to `approvalCategory`